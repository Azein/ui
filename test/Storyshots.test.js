const initStoryshots = require('storyshots').default

initStoryshots()
