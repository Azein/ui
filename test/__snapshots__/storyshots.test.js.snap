exports[`Storyshots Buttons Button 1`] = `
<button
  className="hpTakF">
  Авиабилеты
</button>
`;

exports[`Storyshots Buttons Button 2222 1`] = `
<button
  className="hpTakF">
  12345
</button>
`;
